src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Resources.xaml

```diff
@@ -34,6 +34,9 @@
   <nuget:BooleanToVisibilityConverter
     Inverted="True"
     x:Key="InvertedBooleanToVisibilityConverter" />
+  <nuget:BooleanToVisibilityConverter
+    NonVisualState="Hidden"
+    x:Key="NonVisualStateHiddenBooleanToVisibilityConverter" />
```

Comment:
BooleanToHiddenConverter is what I'd call this. Or something a bit more concise.

BooleanToHiddenInvertedCoverter is what i'd call it if you also inverted it.

Reply:
Not a fan of these renames... if this is a significant concern, I can create an issue to rename it in one of my upcoming PRs.

I'm not sure the benefit of being concise with this Key?